### DISCUSSION TITLE
Orientation from rotation

### DISCUSSION THREAD

**Discussion by CraftedRO** [TrinityCore/TrinityCore#28268] [Q&A] [answered]
I used [this calculator](http://www.andre-gaschler.com/rotationconverter/) to check if we can get orientation from rotation (as Shauren told in an issue) and indeed seems like result for some data are almost similars to what we have in some database fields but there are also some different results from our database now my question is :

How do I know which data is wrong (without manual checking ingame kek) in our database  after getting different result from calculator orientation value or rotation2 / rotation3 ?
for example below I got similar result but negative -0,8028466 while in database we got 0,802847

![Capturerot](https://user-images.githubusercontent.com/24683355/191556023-17a7ebfb-e169-4265-b28a-bd2d3dfeca59.PNG)

**Comment by funjoker** [comment, marked as answer]
it MIGHT be possible, that parser back then messed the signess up

**Reply by CraftedRO** [reply]
well I found few of them [here](https://wow.tools/dbc/?dbc=gameobjects&build=3.4.0.45613#page=1) and [converted them to sql](https://gist.github.com/CraftedRO/c3dd3b3d43d59e732635ca6fe0666a7a) but I still have to calculate orientation for all of them , they seems a lot better than what tc have but they won't pull since we don't know if those data are sniffed or whatever.

**Reply by funjoker** [reply]
Talked to shauren, when rotation was implemented. Some rotation data got corrupted.
